src/EnergyPlus/UnitarySystem.cc

```diff
@@ -4018,20 +4015,32 @@ namespace UnitarySystems {
             }
         }
 
+        if (AirLoopNumber == 0 && !ZoneEquipmentFound &&
```

Comment:
Move this up from below so it executes before the possible `return` below in line 4039.